### PR TITLE
Add response headers to gateways logs

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

File: openapi/components/schemas/TransactionGatewayLog.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ properties:
   response:
     description: The response body.
     type: string
+  responseHeaders:
+    description: The response headers
```
Comment (ashkarpetin) [inline openapi/components/schemas/TransactionGatewayLog.yaml:19]:
```suggestion
    description: The response headers.
```

**Comment by slavcodev** [comment]
@Rebilly/php please review

**Comment by slavcodev** [comment]
@Rebilly/js please review

**Comment by slavcodev** [comment]
@Rebilly/php please review

**Comment by slavcodev** [comment]
@Rebilly/js please review

**Review by colinbird — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**